src/fetch-info.js

```diff
@@ -68,6 +68,19 @@ async function useLastInfoForDiscontinuedSpecs(specs) {
   return results;
 }
 
+function catchAndFallbackOnExistingData(fn) {
+  return function(spec) {
+    return fn(spec).catch(err => {
+      if (spec.__last) {
+	// TODO: log crawl error?
```

Comment:
```suggestion
	// TODO: log crawl error more visibly?
	console.error(`Failed to fetch info on ${spec.url} (${err}), reusing existing data`);
```
This adds console logging; I do agree we need something more robust to detect these situations, but maybe that's a sufficient starting point.